appi.adoc

```diff
@@ -0,0 +1,397 @@
+﻿[[appendix-examples-groups, Appendix I, Recommended Practices and Annotated Examples for Using Groups]]
+
+[appendix]
+== Recommended Practices and Annotated Examples for Using Groups
+
+This appendix contains recommended practices for using netCDF-4 groups in CF compliant data, as well as annotated examples of existing use cases for representing data using groups.
+
+=== Recommended practices
+
+. The use of attributes attached to a group in order to store metadata normally attached directly to variables is discouraged.
+This includes, for example, replacing per-variable attributes like `_FillValue`, `scale_factor`, `valid_min`, with group-level equivalents.
+Although group attributes might be more concise, it is likely to create problems with downstream software and to reduce interoperability.
```

Comment:
This actually sounds dangerous! It is very likely that different variables in the group would have different `scale_factor` values, even assuming they had all been scaled. There are a number of named attributes that do not describe the physical nature of variables, but rather describe the encoding of a data array. These (which include the three that you have named) shouldn't these be specifically prohibited from being set on the group?  What do you think?

Reply:
@davidhassell 
The recommendation _discourages_ using a group-level `scale_factor` to prevent the situation you mention. Perhaps you misread it or I do not understand your concern. Or are you saying the practice should be "forbidden", not merely discouraged?

Reply:
Left as-is for the purpose of minimising changes at this stage.